tests/test_ops/test_merge_cells.py

```diff
@@ -56,11 +56,28 @@ def test_resize_methods():
             merge_cell_out = merge_cell._resize(inputs_x, target_size)
             gt_out = F.interpolate(inputs_x, size=target_size, mode=method)
             assert merge_cell_out.equal(gt_out)
-
-    target_size = (64, 64)  # resize to a smaller size
+    """resize to a smaller size by which the inputs_x can't be divisible.
+    And There're different downsample scale on dim H & dim W.
+    """
```

Comment:
This should be comment like `# resize to a smaller...` but not docstring starts with """